test/test_ops.py

```diff
@@ -1,374 +1,415 @@
+    # AllowedOpInfoEntry('linalg.svdvals'),  # Slice dim size 1 greater than dynamic slice dimension: 0
+    # AllowedOpInfoEntry('lu'),  # Slice dim size 1 greater than dynamic slice dimension: 0
+    # AllowedOpInfoEntry('std_mean'),
+    # AllowedOpInfoEntry('sum'),
+    # AllowedOpInfoEntry('mean'),
+    # AllowedOpInfoEntry('topk'),
+    # AllowedOpInfoEntry('prod'),
+    # AllowedOpInfoEntry('sin'),
+    # AllowedOpInfoEntry('sinh'),
+    # AllowedOpInfoEntry('__rdiv__'),
+    # AllowedOpInfoEntry('__rmod__'),
+    # AllowedOpInfoEntry('std'),
+    # AllowedOpInfoEntry('tan'),
+    # AllowedOpInfoEntry('tanh'),
+    # AllowedOpInfoEntry('expm1'),
+    # AllowedOpInfoEntry('reciprocal'),
+    # AllowedOpInfoEntry('rsqrt'),
+    # AllowedOpInfoEntry('sqrt'),
+    # AllowedOpInfoEntry('linalg.pinv'),
+    # AllowedOpInfoEntry('eig'),
+    # AllowedOpInfoEntry('svd'),
+    # AllowedOpInfoEntry('pinverse'),
+    # AllowedOpInfoEntry('gather'),
+    # AllowedOpInfoEntry('index_fill'),
+    # AllowedOpInfoEntry('index_copy'),
+    # AllowedOpInfoEntry('index_select'),
+    # AllowedOpInfoEntry('index_add'),
+    # AllowedOpInfoEntry('__getitem__'),
+    # AllowedOpInfoEntry('sort'),
+    # AllowedOpInfoEntry('scatter'),
+    # AllowedOpInfoEntry('scatter_add'),
+    # AllowedOpInfoEntry('repeat'),
+    # AllowedOpInfoEntry('squeeze'),
+    # AllowedOpInfoEntry('tile'),
+    # AllowedOpInfoEntry('triangular_solve'),  # Slice dim size 1 greater than dynamic slice dimension: 0
+    # AllowedOpInfoEntry('var'),
+    # AllowedOpInfoEntry('logsumexp'),
+    # AllowedOpInfoEntry('transpose'),
+    # AllowedOpInfoEntry('to_sparse'),
+    # AllowedOpInfoEntry('sigmoid'),
+    # AllowedOpInfoEntry('erf'),
+    # AllowedOpInfoEntry('erfc'),
+    # AllowedOpInfoEntry('erfinv'),
+    # AllowedOpInfoEntry('norm'),
+    # AllowedOpInfoEntry('t'),
+    # AllowedOpInfoEntry('logdet'), xla::lodget does not handle empty input
+    # AllowedOpInfoEntry('qr'),  # Slice dim size 1 greater than dynamic slice dimension: 0
+
+    # Failed on CUDA CI only (investigate)
+    # app.circleci.com/pipelines/github/pytorch/xla/9088/workflows/2d59c649-db2b-4384-921e-5e43eba1b51a/jobs/17875
+    # AllowedOpInfoEntry('index_put'),
+
+    # Worked locally (but failing on CI both CPU and CUDA)
+    # app.circleci.com/pipelines/github/pytorch/xla/9130/workflows/71c74f3d-1735-4328-81b5-784d6e6744da/jobs/17998
+    # AllowedOpInfoEntry('var_mean'),
+    # AllowedOpInfoEntry('pow'), # for int64 don't work, likely rounding issue
+    # AllowedOpInfoEntry('__rpow__'),
+
+    # In theory, this should work.
+    # However, the problem is the way we prepare the reference (CPU) inputs:
+    # we clone them. If they were a view, they are not anymore.
+    #
+    # AllowedOpInfoEntry('as_strided', 'partial_views'),
+)
+
+allowed_fallback_opinfo = get_allowed_ops_map(
+    AllowedFallbackOpInfoEntry(
```

Comment:
I wonder how these ops are selected. 
Also, there are existing ops that can potentially falls back such as `nonzero`, do you also plan to test it?

Reply:
> I wonder how these ops are selected.

I selected the fallback operations that show-up when running torchbench.

>  Also, there are existing ops that can potentially falls back such as `nonzero`, do you also plan to test it?

Ah, right. That one does appear in torchbench, but I think I forgot about that one. I will add it.